### ISSUE TITLE
BMT 1 - Fix heading issues flagged by the new Cypress Heading rules launching April 30

### ISSUE THREAD

**Issue by bacitracin** [department-of-veterans-affairs/va.gov-team#108007]
## Issue

New Cypress Axe rules for headings are coming and will be put into effect April 30. [Slack thread](https://dsva.slack.com/archives/C03R5SBELQM/p1742920275472879)

I've created tickets for heading issues I've noticed while manually running Axe Devtools scans, but this ticket is to collect the heading issues we MUST fix because they lead to breaking Cypress tests.

Note: Ran tests on the following folders, let me know if there are any I missed:
- src/applications/claims-status/tests/e2e
- src/applications/letters/tests
- src/applications/rated-disabilities/tests/e2e
- src/applications/disability-benefits/view-payments/tests/e2e
- src/applications/appeals

### Claims Status Tool
- [x] 00.claims-list-empty.cypress.spec.js <img width="1480" alt="Image" src="https://github.com/user-attachments/assets/1df5ecd8-2bf3-4031-9c99-e1f2c7a1254a" />

- [x] 07.appeal-status.cypress.spec.js <img width="1468" alt="Image" src="https://github.com/user-attachments/assets/2fe93f61-5a4f-48d2-a6a2-d2c88132b4ef" />

### VA Benefit Letters

- [x] 01-authed-lighthouse.cypress.spec.js <img width="1510" alt="Image" src="https://github.com/user-attachments/assets/a7d31282-da6a-4d11-8a8c-c33bf576ba2a" />
- [x] 01-authed.cypress.spec.js <img width="1506" alt="Image" src="https://github.com/user-attachments/assets/6e84b8a3-b3c7-445e-a857-5cb7b5d21a16" />
- [x] 02-keyboard-only.cypress.spec.js <img width="1512" alt="Image" src="https://github.com/user-attachments/assets/1c0f603b-0aaa-435f-84db-e122a006170a" />

### Rated Disabilities
Note: all tests passing, no heading violations to fix
